sebs/aws/s3.py

```
@@ -46,11 +62,29 @@ def __init__(
 
     def correct_name(self, name: str) -> str:
+        """
+        Return the corrected bucket name (no correction needed for S3).
+
+        :param name: Original bucket name.
+        :return: Corrected bucket name.
+        """
         return name
 
     def _create_bucket(
         self, name: str, buckets: List[str] = [], randomize_name: bool = False
```

Ruff (0.11.9)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)